hal/transport/RFM69/driver/new/RFM69_new.h

```diff
@@ -150,7 +150,7 @@
 
 // CSMA settings
 #if !defined(MY_RFM69_CSMA_LIMIT_DBM)
-#define MY_RFM69_CSMA_LIMIT_DBM             (-95)			//!< upper RX signal sensitivity threshold in dBm for carrier sense access
+#define MY_RFM69_CSMA_LIMIT_DBM             (-85)			//!< upper RX signal sensitivity threshold in dBm for carrier sense access
```

Comment:
Could we document why -85 is better than -95?

Reply:
It will improve traffic in case of very noisy power supply (usb ftdi adapter with ultra low power ldo for example :)).  Of course it increases the probability of collisions in case of very long range. We can omit this but put the comment.

Reply:
I just remembered your comment in 
https://forum.mysensors.org/post/91435
I suggest we go with -85 as default value, since your tests show that -95 is too optimistic.